README.md

```diff
@@ -4,18 +4,15 @@
 
 ## What is this?
 
-This repository allows you to quickly install Adminer database manager into a [Ddev](https://ddev.readthedocs.io) project using just `ddev get ddev/ddev-adminer`.
+This repository allows you to quickly install the Adminerevo fork of the adminer database manager into a [Ddev](https://ddev.readthedocs.io) project using just `ddev get ddev/ddev-adminer`.
 
 Adminer is a full-featured database management tool written in PHP. This service
 currently ships the [official adminer container](https://hub.docker.com/_/adminer)
 with no _external_ plugins. It contains all official plugins and themes and allows
 to easily chose one by editing the `docker-compose.adminer.yaml` file after
 installation.
 
-This currently supports:
-
-* MySQL / MariaDB
-* PostgreSQL
+Adminer is available for MySQL, MariaDB, PostgreSQL, SQLite, MS SQL, Oracle, Elasticsearch and MongoDB.
```

Comment:
```suggestion
AdminerEvo is available for MySQL, MariaDB, PostgreSQL, SQLite, MS SQL, Oracle, Elasticsearch and MongoDB.
```

Reply:
How about "Adminer works with"

Moving to mention of AdminerEvo may end up adding confusion. I know we're changing the technology behind this. Should we just mention that in the README and then still use regular adminer terminology?

Reply:
I like that idea, to update the README to reflect that the adminer add-on uses adminerEvo

Reply:
I changed the readme